### PR TITLE
Added ProfileStoreTest

### PR DESCRIPTION
**Pull request by isabellasalvi**

### CODE REVIEW

File: src/main/java/codeu/model/store/basic/ProfileStoreTest.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito; 
+
+public class ProfileStoreTest {
+
+  private ProfileStore profileStore;
+  private PersistentStorageAgent mockPersistentStorageAgent;
+
+  private final Profile PROFILE_ONE =
```
Comment (UnitingDust) [inline src/main/java/codeu/model/store/basic/ProfileStoreTest.java]: The current Profile class doesn't have a field related to time. Was this a change that you made?

File: src/main/java/codeu/model/store/basic/ProfileStore.java [hunk 2]
```diff
@@ -70,14 +70,29 @@ public void loadTestData() {
...
   /**
    * Access the Profile object with the given UUID.
    *
    * @return null if the UUID does not match any existing Profile.
    */
-  public Profile getProfile(UUID ID) {
+  public Profile getProfile(UUID id) {
     for (Profile profile : profiles) {
-      if (profile.getUserID().toString().equals(ID.toString())) {
+      if (profile.getId().equals(id)) {
```
Comment (UnitingDust) [inline src/main/java/codeu/model/store/basic/ProfileStore.java]: Are you referring to "public UUID getUserID()" method of the Profile class?

File: src/test/java/codeu/model/store/basic/ProfileStoreTest.java [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+    Assert.assertNull(resultProfile);
+  }
+
+  @Test
+  public void testAddProfile() {
+    Profile inputProfile = new Profile(UUID.randomUUID(), "description");
+
+    profileStore.addProfile(inputProfile);
+    Profile resultProfile = profileStore.getProfile("description");
```
Comment (UnitingDust) [inline src/test/java/codeu/model/store/basic/ProfileStoreTest.java]: getProfile() method takes in a UUID to get the profile. I think you might want to create a UUID variable such as
UUID id = UUID.randomUUID();

Then you pass that variable into the constructor to create a new profile. 
Profile inputProfile = new Profile(id, "description");

Finally you use profileStore.getProfile(id) to get that profile.

**Comment by ryanluo** [comment]
Isabella, can you set up a meeting with Angel to sort out the file conflicts? Please ping this thread when that's all sorted out. Thanks!

**Pull request closed without merging**